metrics/ngapp/src/app/metrics-page.component.html

```diff
@@ -3,6 +3,15 @@
     Metrics
     <div class="spinner spinner-inverse spinner-sm" *ngIf="loading"></div>
   </div>
+  <div class="alert alert-info alert-dismissable" *ngIf="showGrafanaAlert">
+    <button type="button" class="close" data-dismiss="alert" aria-hidden="true" (click)="showGrafanaAlert=false">
+      <span class="pficon pficon-close"></span>
+    </button>
+    <span class="pficon pficon-info"></span>
+    <strong>Intended for test and debug!</strong> For more capable dashboards display we suggest that you take a look at our
```

Comment:
ok, maybe my english knowledge reached a limit here :) I intended to say "dashboard with more capabilities" indeed.